Jenkinsfile

```diff
@@ -22,19 +22,13 @@ pipeline {
 						}
 					},
 					"linux": {
```

Comment:
Rename this to build Please

Reply:
1. You want to rename "linux" to "build"? What about "windows" then?
2. There is no linting stage. I am not sure what you mean.